tests/drillhole_v4_0_test.py

```diff
@@ -630,6 +630,17 @@ def test_remove_drillhole_data(tmp_path):
         assert "my_log_values/" not in well.get_entity_list()
         assert workspace.get_entity("well copy")[0] is None
 
+    with Workspace(h5file_path, version=2.0) as workspace:
+        well = workspace.get_entity("well")[0]
+        all_data = [well.get_entity(name)[0] for name in well.get_data_list()]
+        all_data = [data for data in all_data if data.allow_delete]
+        well.remove_children(all_data)
+
+        assert len(well.property_groups) == 0
+
+        well = workspace.get_entity("Number 3")[0]
+        well.remove_children(well.property_groups[0])

```

Comment:
don't you want to assert it has been removed?

Reply:
yep, good point